### PR TITLE
Bugfix/df 1731 improve kofax exporter logging

### PR DESCRIPTION
**Pull request by seco-mgabor**

### CODE REVIEW

File: app-kofax-exporter/src/test/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/xml/KofaxXmlGeneratorTest.java [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 
 public class KofaxXmlGeneratorTest {
 
-	private final String CLOSING_XML_TAGS = "</Pages></Document></Documents></Batch></Batches></ImportSession>";
+	private final static String CLOSING_XML_TAGS = "</Pages></Document></Documents></Batch></Batches></ImportSession>";
```
Comment (dvoracek-martin) [inline app-kofax-exporter/src/test/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/xml/KofaxXmlGeneratorTest.java]: Why would you have it static?

Comment (seco-mgabor) [reply app-kofax-exporter/src/test/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/xml/KofaxXmlGeneratorTest.java]: It looks like a constant to me.

Comment (dvoracek-martin) [inline app-kofax-exporter/src/test/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/xml/KofaxXmlGeneratorTest.java]: Why this need to be static?

Comment (seco-mgabor) [reply app-kofax-exporter/src/test/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/xml/KofaxXmlGeneratorTest.java]: It's a constant.

File: app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/ExportHandler.java [hunk 2]
```diff
@@ -52,25 +50,27 @@ public void handleDomainEvents(ExportHandlerData exportHandlerData) throws IOExc
 		final String name = exportHandlerData.getName();
 		final Path documentPath = workingDirectory.resolve(name + resolveExtension(exportHandlerData.getDocumentMimeType()));
 		final Path xmlPath = workingDirectory.resolve(name + ".xml");
+		LOGGER.info("Start generating XML file...");
```
Comment (pavax) [inline app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/ExportHandler.java]: remove the logging here as it is missleading you could however add the following

```java
LOGGER.info("Start to export data for event: {}  Id: {}", exportHandlerData.getEvent().getClass().getSimpleName(), name)
```

File: app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/ExportHandler.java [hunk 3]
```diff
@@ -52,25 +50,27 @@ public void handleDomainEvents(ExportHandlerData exportHandlerData) throws IOExc
...
 			IOUtils.copy(documentContent.getContent(), documentOutputStream);
 			String pathToPdf = xmlPath.getParent().relativize(documentPath).toString();
 			KofaxData kofaxData = this.kofaxDataFactory.extract(exportHandlerData.getEvent(), pathToPdf);
+			LOGGER.info("Source document {}", documentPath.getFileName());
+			LOGGER.info("XML output document {}", xmlPath.getFileName());
 			this.xmlGenerator.generate(kofaxData, xmlOutputStream);
-		} catch (Exception e) {
-			LOGGER.debug("Problem while exporting Documents: {}, {} to KOFAX.", documentPath.getFileName(), xmlPath.getFileName());
+		} catch (IOException e) {
+			LOGGER.error("Problem while generating export documents: {}, {} for KOFAX.",
```
Comment (pavax) [inline app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/ExportHandler.java]: good catch -> thanks

**Comment by seco-mgabor** [comment]
Thanks @pavax for the detailed code review. I've applied the changes you've suggested.
I hope it's OK now.

**Pull request merged**